content/applications/productivity/iot/config/connect.rst

```diff
@@ -1,81 +1,145 @@
+Go to the :menuselection:`IoT` App and click on :guilabel:`Connect` on the
+:abbr:`IoT (Internet of Things)` IoT Boxes page.
 
-.. image:: connect/connect02.png
+.. image:: connect/connect09.png
    :align: center
+   :alt: Connecting an IoT box to the Odoo database.
 
-Follow the steps to connect your IoT Box.
+Follow the steps to connect the IoT Box either via wired ethernet connection or via WiFi.
 
-.. image:: connect/connect03.png
+.. image:: connect/connect-iot-box.png
    :align: center
+   :alt: Connection steps for a wired connection or WiFi connection.
 
-Ethernet Connection
-===================
+.. note::
+   Ensure the :abbr:`IoT (Internet of Things)` box is flashed with the most up-to-date
+   :abbr:`IoT (Internet of Things)` image.
 
-#. Connect to the IoT Box all the devices that have to be connected with
-   cables (ethernet, usb devices, etc.).
+Ethernet connection
+===================
 
-#. Power on the IoT Box.
+#. Connect all the devices that have to be connected with cables to the
+   :abbr:`IoT (Internet of Things)` box (ethernet, :abbr:`USB (Universal Serial Bus)` devices,
+   etc.).
 
-#. Read the Pairing Code from a screen or a receipt printer connected to the IoT Box.
+#. Power on the :abbr:`IoT (Internet of Things)` Box.
 
-   .. image:: connect/connect04.png
+#. Read the :guilabel:`Pairing Code` from a screen or a receipt printer connected to the
+   :abbr:`IoT (Internet of Things)` Box.
 
-#. Input the Pairing Code and click on the Pair button.
+   .. note::
+      If no screen is attached to the :abbr:`IoT (Internet of Things)` box then the pairing code can
+      be accessed from the :abbr:`IoT (Internet of Things)` box admin console by clicking on
+      :guilabel:`POS Display`. For instructions on how to access the admin console, visit
+      :ref:`iot_connect/token`
 
-.. note::
-   Recent changes in modern web browsers forced us to modify the connection wizard.
-   If your screen is different from the screenshots, make sure that the
-   ``iot_pairing`` module is installed.
+#. Input the :guilabel:`Pairing Code` on the :abbr:`IoT (Internet of Things)` app of the database
+   and click on the :guilabel:`Pair` button.
 
-WiFi Connection
+WiFi connection
 ===============
 
-#. Power on the IoT Box
+#. Connect all the devices that have to be connected with cables to the
+   :abbr:`IoT (Internet of Things)` box (ethernet, :abbr:`USB (Universal Serial Bus)` devices,
+   etc.).
 
-#. Copy the token
+#. Power on the :abbr:`IoT (Internet of Things)` Box.
 
-   .. image:: connect/connect05.png
+#. Copy the :guilabel:`Token` from the :guilabel:`WiFi connection` section in
+   :menuselection:`IoT` app of the Odoo database.
 
-#. Connect to the IoT Box WiFi Network (make sure there is no ethernet
-   cable plugged in your computer).
+#. Connect to the :abbr:`IoT (Internet of Things)` Box WiFi Network (make sure there is no ethernet
+   cable plugged into the computer).
```

Comment:
might be worth to clarify that "the computer" is the device that will do the wifi connection with the IoT box. It's a bit uncleared IMHO